code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +165,26 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone

```

Comment:
Don't forget to override Bumped() and have it just return otherwise people can spam the sounds by running over it repeatedly